app/feature.go

```diff
@@ -0,0 +1,97 @@
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	ExportFeatureFlag = "enable-export"
+)
+
+func GetFeatureFlagConfigPath() string {
+	return ConfigDirFlag.Value + "/feature.json"
+}
+
+func GetFeatureFlagConfig(featureFlagConfigPath string) (map[string]bool, error) {
+	// create config file if not exist
+	if _, err := ioutil.ReadFile(featureFlagConfigPath); err != nil {
+		if err := ioutil.WriteFile(featureFlagConfigPath, []byte("{}"), 0644); err != nil {
+			return nil, err
+		}
+	}
+
+	content, err := ioutil.ReadFile(featureFlagConfigPath)
+	if err != nil {
+		return nil, err
+	}
+
+	var jsonData map[string]bool
+	err = json.Unmarshal(content, &jsonData)
+	if err != nil {
+		return nil, err
+	}
+
+	return jsonData, nil
+}
+
+func NewFeatureCommand() (CommandOut, error) {
+	return CommandOut{
+		Command: &cli.Command{
+			Name:    "feature",
+			Aliases: []string{"f"},
+			Usage:   "feature commands",
+			Hidden:  true,
+			Subcommands: []*cli.Command{
+				{
+					Name:    "toggle-export",
```

Comment:
actually looking at this again i wonder if it's better to offer an enable/disable option per feature flag?

in the future a feature flag might also contain a value--so i think structuring it as key/value makes sense to me.

Reply:
toggle-export will be working in that way. I'm flipping the value all the time. Can find the detail on top.